src/common/bpf/counters.rs

```diff
@@ -50,32 +51,49 @@ impl<'a> Counters<'a> {
         }
     }
 
-    pub fn refresh(&mut self, now: Instant, elapsed: f64) {
+    pub fn refresh(&mut self, elapsed: f64) {
         for value in self.values.iter_mut() {
```

Comment:
No strong opinion, but since this is perfomance-sensitive: would a `slice.fill` be faster here?